src/commands/sign.ts

```diff
@@ -1,69 +1,44 @@
 import { Argv } from "yargs";
+import { withWalletOption } from "./shared";
 import signale from "signale";
-import { sign, Output } from "../implementations/sign";
-import { isDir } from "../implementations/utils/disk";
+import { sign, SignCommand, supportedAlgorithms } from "../implementations/sign";
 
 export const command = "sign <raw-documents-path>";
 
 export const describe = "Sign document(s) and appends proof block";
 
 export const builder = (yargs: Argv): Argv =>
-  yargs
-    .positional("raw-documents-path", {
-      description: "Directory containing the unissued raw documents or a single raw document file",
-      normalize: true,
-      type: "string",
-    })
-    .option("output-file", {
-      alias: "of",
-      description: "Write output to a file. Only use when <wrapped-documents-dir> is a document",
-      type: "string",
-      conflicts: "output-dir",
-    })
-    .option("output-dir", {
-      alias: "od",
-      description: "Write output to a directory",
-      type: "string",
-      conflicts: "output-file",
-    })
-    .option("private-key", {
-      description: "Private key to sign document(s) with",
-      type: "string",
-    })
-    .option("public-key", {
-      description: "Public key, added to the document, used to verify the signature is valid",
-      type: "string",
-    });
-
-export interface SignCommand {
-  rawDocumentsPath: string;
-  outputDir?: string;
-  outputFile?: string;
-  privateKey: string;
-  publicKey: string;
-}
+  withWalletOption(
```

Comment:
Not a big deal but I find it slightly weird because of `encrypted-wallet-path` option. Not sure if it makes sense for other did than eth ?

Reply:
I just noticed that the option is not used, can use `withPrivateKeyOption`

Reply:
Yep, its unintended, changing to withPrivateKeyOption